index.html

```diff
@@ -97,8 +97,20 @@ <h3 class="info-title">Multi-Post Stories</h3>
                  <li class="lang-box">Bootstrap</li>
```

Comment:
Please use linter for HTML, CSS, and JavaScript from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) linter config.

Reply:
To fix the linters checks you can add the linter config to the linters.yml file then close the pull request and then reopen it again. This will restart the config for the linter checks.

Reply:
Thanks for the feedback.